anndata/tests/test_concatenate.py

```diff
@@ -98,11 +98,21 @@ def fill_val(request):
     return request.param
 
 
-@pytest.fixture(params=[0, 1])
-def axis(request) -> Literal[0, 1]:
+@pytest.fixture(params=["obs", "var"])
+def axis_name(request) -> Literal["obs", "var"]:
     return request.param
 
 
+@pytest.fixture
+def axis(axis_name) -> Literal[0, 1]:
+    return 0 if axis_name == "obs" else 1
+
+
+@pytest.fixture
+def alt_axis_name(axis_name) -> Literal["obs", "var"]:
+    return "var" if axis_name == "obs" else "obs"
+
```

Comment:
Instead of having two parameters (which function as one) can we just have one, then derive the value in the test?

Reply:
That’s how it was before, but I found stuff like `("uns", "var)[axis]` to be too cryptic and mistakes too easily missable.

That’s why I switched to a centralized, clean alternative.